examples/api/src/index.ts

```diff
@@ -343,7 +343,6 @@ export async function runServer() {
     urlAdapter: new ExampleURLAdapter({websiteURL}),
     playground: true,
     introspection: true,
-    tracing: true,
```

Comment:
Removed as now not in Apollo anymore, could be brought back with graphql-tracing but I don't think we should have that for now